docs/contributor/rhel4edge_iso.md

```diff
@@ -299,14 +290,14 @@ The contents of the archive can be used for serving ostree updates:
 > `-embed_containers` arguments to customize the edge commit archive contents.
 
 Copy the produced `microshift-0.0.1-commit.tar` archive to the
-MicroShift server and unpack it at the `/var/lib/ostree-server-local` directory,
+MicroShift server and unpack it at the `/var/lib/ostree-local` directory,
```

Comment:
Is that path something known to rpm-ostree, or is that a path name we're suggesting that has no special meaning?

Reply:
It's a path that we're suggesting.

Reply:
@dhellmann , please unhold the PR if you think the current approach is acceptable.